src/library-authoring/LibraryAuthoringPage.jsx

```diff
@@ -75,55 +93,82 @@ const LibraryAuthoringPage = () => {
-            element={<LibraryCollections />}
-          />
-          <Route
-            path="*"
-            element={<NotFoundAlert />}
-          />
-        </Routes>
+      <Container>
+        <Row>
+          <Col>
+            <Header
+              number={libraryData.version.toString()}
+              title={libraryData.title}
+              org={libraryData.org}
+              contentId={libraryId}
+              isLibrary
+            />
+            <Container size="xl" className="p-4 mt-3">
+              <SubHeader
+                title={<SubHeaderTitle title={libraryData.title} />}
+                subtitle={intl.formatMessage(messages.headingSubtitle)}
+                headerActions={[
+                  <Button
+                    iconBefore={Add}
+                    variant="primary rounded-0"
+                    onClick={() => dispatch(openAddContentSidebar())}
+                    disabled={!libraryData.canEditLibrary}
+                  >
+                    {intl.formatMessage(messages.newContentButton)}
+                  </Button>,
+                ]}
+              />
+              <SearchField
+                value={searchKeywords}
+                placeholder={intl.formatMessage(messages.searchPlaceholder)}
+                onSubmit={(value) => setSearchKeywords(value)}
+                onChange={(value) => setSearchKeywords(value)}
+                className="w-50"
+              />
+              <Tabs
+                variant="tabs"
+                activeKey={tabKey}
+                onSelect={handleTabChange}
+                className="my-3"
+              >
+                <Tab eventKey={TAB_LIST.home} title="Home" />
+                <Tab eventKey={TAB_LIST.components} title="Components" />
+                <Tab eventKey={TAB_LIST.collections} title="Collections" />
+              </Tabs>
+              <Routes>
+                <Route
+                  path={TAB_LIST.home}
+                  element={<LibraryHome libraryId={libraryId} filter={{ searchKeywords }} />}
+                />
+                <Route
+                  path={TAB_LIST.components}
+                  element={<LibraryComponents libraryId={libraryId} filter={{ searchKeywords }} />}
+                />
+                <Route
+                  path={TAB_LIST.collections}
+                  element={<LibraryCollections />}
+                />
+                <Route
+                  path="*"
+                  element={<NotFoundAlert />}
+                />
+              </Routes>
+            </Container>
+            <StudioFooter />
+          </Col>
+          {showSidebar && (
+            <Col xs={6} md={4} className="box-shadow-left-1">
+              <LibrarySidebar />
+            </Col>
+          )}
+        </Row>
       </Container>
-      <StudioFooter />
+      <Toast
+        show={showToast}
+        onClose={() => dispatch(closeToast())}
+      >
+        {toastMessage}
+      </Toast>
```

Comment:
This is generating a warning:
```
Warning: Failed prop type: The prop `children` is marked as required in `Toast`, but its value is `null`.
```

```suggestion
      { toastMessage && <Toast
        show={showToast}
        onClose={() => dispatch(closeToast())}
      >
        {toastMessage}
      </Toast>}
```
or default `toastMessage` to `''`